server/datastore/mysql/policies.go

```diff
@@ -77,18 +77,41 @@ func (ds *Datastore) RecordPolicyQueryExecutions(ctx context.Context, host *flee
 		strings.Join(bindvars, ","),
 	)
 
-	return ds.withRetryTxx(ctx, func(tx sqlx.ExtContext) error {
+	err := ds.withRetryTxx(ctx, func(tx sqlx.ExtContext) error {
 		_, err := tx.ExecContext(ctx, query, vals...)
 		if err != nil {
 			return errors.Wrapf(err, "insert policy_membership (%v)", vals)
 		}
 
-		_, err = tx.ExecContext(ctx, `UPDATE hosts SET policy_updated_at = ? WHERE id=?`, host.PolicyUpdatedAt, host.ID)
```

Comment:
Q: Was using `host.PolicyUpdatedAt` instead of `updated` a bug?

Reply:
No because we updated that to the same value as updated before we call this.

Reply:
The change is just for consistency.